docs/Getting_started.rst

```diff
@@ -10,8 +10,9 @@ Getting started
 ===============
 
 **Before starting:** get your Mila cluster account ready. Typically, your
```

Comment:
Side note: I  wonder if we should reference the Mila IT Onboarding instead of  `get your Mila cluster account ready`. This would add a link to the Intranet, which is not accessible from the Web. However this could be less ambiguous for newcomers.

@tesfaldet @lavoiems @vict0rsch @mila-iqia/orion any thoughts?

Reply:
#71